Sources/Logging/Protocols/Logger.swift

```diff
@@ -8,42 +8,36 @@
 
 protocol Logger {
 
-    // MARK:- Destination Management
-
-    func register(_ destination: LogDestination)
-    func unregister(_ destination: LogDestination)
-    func removeAllDestinations()
-
     // MARK:- Logging
 
     func verbose(_ message: @autoclosure () -> String,
                  file: StaticString,
-                 function: String,
+                 function: StaticString,
                  line: UInt)
 
     func debug(_ message: @autoclosure () -> String,
                file: StaticString,
-               function: String,
+               function: StaticString,
                line: UInt)
 
     func info(_ message: @autoclosure () -> String,
               file: StaticString,
-              function: String,
+              function: StaticString,
               line: UInt)
 
     func warning(_ message: @autoclosure () -> String,
                  file: StaticString,
-                 function: String,
+                 function: StaticString,
                  line: UInt)
 
     func error(_ message: @autoclosure () -> String,
                file: StaticString,
-               function: String,
+               function: StaticString,
                line: UInt)
     
     func log(level: Log.Level,
```

Comment:
Well, it isn't strictly necessary. I thought it would be better to leave that *generic* to write something into the log open, but you're right. I guess there won't be much cases, if any, this generic method is of any use. Made it private.